src/contracts/user-deployer-whitelist.ts

```diff
@@ -92,4 +116,43 @@ export class UserDeployerWhitelist extends Contract {
       address: Address.UnmarshalPB(deployerContract.getContractAddress()!)
     }))
   }
+
+  /**
+   * @param TierID tierId.
+   * @returns Tier object containing TierDetails.
+   */
+  async getTierInfoAsync(tierId: TierID): Promise<ITier> {
+    const req = new GetTierInfoRequest()
+    req.setId(tierId)
+    const result = await this.staticCallAsync(
+      'GetTierInfo',
+      req,
+      new GetTierInfoResponse()
+    )
+    return {
+      tierId: result.getTier()!.getTierId(),
+      fee: unmarshalBigUIntPB(result.getTier()!.getFee()!),
+      name: result.getTier()!.getName()
+    }
+  }
+
+  /**
+   * Allows to modify TierInfo, caller can only be owner set in init.
+   *
+   * @param TierID tierId
+   * @param fee
+   * @param name
+   */
+  setTierInfoAsync(tierId: TierID, fee: BN, name: string): Promise<void> {
+    if (fee.cmp(new BN(0)) <= 0) {
+      throw Error('whitelisting fees should be greater than zero')
+    }
+    const req = new ModifyTierInfoRequest()
+    req.setId(tierId)
+    req.setFee(marshalBigUIntPB(fee))
+    req.setName(name)
+    return this.callAsync<void>('ModifyTierInfo', req)
```

Comment:
This is now calling a function that doesn't exist.

Reply:
Done.